cpu/atmega_common/include/periph_cpu_common.h

```diff
@@ -177,15 +177,29 @@ typedef struct {
 /** @} */
 
 /**
-￼* @name RTT configuration
-￼* @{
-￼*/
-#define RTT_MAX_VALUE    (0x00FFFFFF)    /* 24-bit timer */
+ * @name RTT configuration
+ * @{
+ */
+#ifdef SCCR0
```

Comment:
```suggestion
#if defined(SCCR0) && !defined(RTT_BACKEND_SC)
#define RTT_BACKEND_SC    (1)
#endif

#if RTT_BACKEND_SC
```

and move it here.

Also `s/#ifdef RTT_BACKEND_SC/#if RTT_BACKEND_SC/g`, `s/#ifdnef RTT_BACKEND_SC/#if RTT_BACKEND_SC == 0/g` so `CFLAGS+=-DRTT_BACKEND_SC=0` works.

Reply:
Please check if it meets expectations

Reply:
Both backends work fine, please squash!